.github/workflows/publish_latest_chart.yml

```diff
@@ -21,6 +21,10 @@ jobs:
       - name: Helm Add Repo
         run: helm repo add chartmuseum https://gitops:${{ secrets.HELM_PASSWORD }}@charts.fluvio.io
       - name: Push Sys Chart
-        run: helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)-latest" --force chartmuseum
+        run: |
+          helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)" --force chartmuseum
+          helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)+$(git rev-parse HEAD)" --force chartmuseum
```

Comment:
We shouldn't need `--force` for this, as we are not overriding.

Reply:
Good point